Library/Formula/slacknimate.rb

```diff
@@ -0,0 +1,24 @@
+class Slacknimate < Formula
+  desc "text animation for Slack messages"
+  homepage "https://github.com/mroth/slacknimate"
+  url "https://github.com/mroth/slacknimate/archive/v1.0.0.tar.gz"
+  sha256 "7d495e2233a379f3ffffb2445f4b41b4306e52049d71f4c4c873786bb4048f45"
+  head "https://github.com/mroth/slacknimate.git"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/mroth"
+    ln_s buildpath, buildpath/"src/github.com/mroth/slacknimate"
+    system "glide", "install"
```

Comment:
I'd like to respectfully push back on this one for the sake of generating discussion.  

As the Go ecosystem is developing, there is an increasing focus on solving the "reproducible build" problem, which `glide` is striving to help address[1].  My issue with using `go_resource` for a formula here is it creates a secondary list of dependencies that is duplication of what is contained in the glide vendored/locked deps in the repo, and could easily get out of a sync.

To be more specific, as a project maintainer, the two specific situations the `go_resource` solution cause me to be concerned about is: 

1. A contributor bumps the package version/sha256sum here, but _does not manually update all the `go_resource` pinning_ to match.  The formula likely continues to compile, and people installing via homebrew now get bottled binaries with an identical version number, but *different program* than people who install packaged binaries produced via makefile/other sources, leading to unpredictable behavior and heisenbugs getting reported to the original project.

2. Similarly, building `--HEAD` will not pick up upstream dependency changes in the deps file, leading to potential unpredictable breakage whenever trunk introduces a new dependency.

I've actually just pushed a new commit that changes the install phase to use the Makefile rather than try to duplicate it, which I think is even more robust.  Take a look at that let me know what you think!